doc/guides/west/manifest.rst

```diff
@@ -271,3 +285,865 @@ repository root.
 
 The pykwalify schema :file:`manifest-schema.yml` in the west source code
 repository is used to validate the manifest section.
+
+.. _west-manifest-import:
+
+Manifest Imports
+****************
+
+You can use the ``import`` key briefly described above to include projects from
+other manifest files in your :file:`west.yml`. This key can be either a
+``project`` or ``self`` section attribute:
+
+.. code-block:: yaml
+
+   manifest:
+     projects:
+       - name: some-project
+         import: ...
+     self:
+       import: ...
+
+You can use this feature to import projects from elsewhere (either another git
+repository, or somewhere else inside your manifest repository) into your own
+:file:`west.yml`.
```

Comment:
I think this should be rephrased so it becomes clearer what is what.
- self import section can be used to load additional files from inside your manifest repo.
- project import can be used to load additional projects described in a git repository manifest file.

Note: should we avoid referring directly to `git` to be prepared for the day when `west` supports non-git projects ?

Reply:
> I think this should be rephrased so it becomes clearer what is what.

Addressed, thanks